src/app/forms/components/contingency-adr-generate-cert/contingency-adr-generate-cert.component.html

```diff
@@ -0,0 +1,28 @@
+<ng-container>
```

Comment:
Do you need an empty ng-container, or is this missing an *ngIf?

Reply:
Think I must have tried something with it and left it in place. I'll remove it